_content/tour/ind/pointers.article

```diff
@@ -1,151 +1,151 @@
-    // Pointer variables are literal types and are declared using *.
+    // increment2 mendeklarasikan fungsi untuk menerima salinan sendiri dari
+    // alamat yang menunjuk ke nilai integer.
+    // Variabel pointer adalah tipe literal dan dideklarasikan menggunakan *.
     func increment2(inc *int) {
 
-        // Increment the caller’s int value through the pointer.
+        // Menambah nilai int pemanggil melalui pointer.
         *inc++
         println(
-            "inc2:\tValue Of[", 
-            inc, "]\tAddr Of[", &inc, 
+            "inc2:\tValue Of[",
+            inc, "]\tAddr Of[", &inc,
             "]\tPoints To[", *inc, "]")
     }
 
-Output:
+Keluaran:
 
     count:  Value Of[ 10 ]  Addr Of[ 0xc000050738 ]
     inc1:   Value Of[ 11 ]  Addr Of[ 0xc000050730 ]
     count:  Value Of[ 10 ]  Addr Of[ 0xc000050738 ]
     inc2:   Value Of[ 0xc000050738 ] Addr Of[ 0xc000050748 ] Points To[ 11 ]
     count:  Value Of[ 11 ]  Addr Of[ 0xc000050738 ]
 
-** Notes
+** Catatan
 
-- Use pointers to share data.
-- Values in Go are always pass by value.
-- "Value of", what's in the box. "Address of" ( & ), where is the box.
-- The ( * ) operator declares a pointer variable and the "Value that the pointer points to".
+- Gunakan pointer untuk berbagi data.
+- Nilai dalam Go selalu dilewatkan dengan nilai.
+- "Nilai" adalah apa yang ada di dalam kotak. "Alamat" ( & ), adalah dimana dimana kotak berada dalam memori.
+- Operator ( * ) mendeklarasikan variabel penunjuk (pointer) dan "Nilai yang ditunjuk oleh penunjuk".
 
 ** Escape Analysis
 
-The algorithm the compiler uses to determine if a value should be constructed
-on the stack or heap is called "escape analysis". The name of the algorithm makes
-it sound like values are constructed on the stack first and then escape (or move)
-to the heap when necessary. This is NOT the case. The construction of a value only
-happens once, and the escape analysis algorithm decides where that will be (stack
-or heap). Only construction on the heap is called an allocation in Go.
-
-Understanding escape analysis is about understanding value ownership. The idea is,
-when a value is constructed within the scope of a function, then that function
-owns the value. From there ask the question, does the value being constructed still
-have to exist when the owning function returns? If the answer is no, the value can
-be constructed on the stack. If the answer is yes, the value must be constructed on
-the heap.
-
-Note: The ownership rule is a good base rule for identifying code that causes
-allocations. However, you must appreciate that escape analysis has flaws that can
-result in non-obvious allocations. Also, the algorithm takes opportunities to
-leverage compiler optimizations to save on allocations.
-
-    // user represents a user in the system.
+Algoritma yang digunakan oleh kompilator (compiler) untuk menentukan apakah sebuah nilai
+harus dibuat di dalam stack atau heap disebut "escape analysis".
```

Comment:
since you're not translating escape analysis from 127 and 130, I think you dont need to translate that in line 18

Reply:
ack